information-system/data-store/access-control.md

```diff
@@ -0,0 +1,101 @@
+
+## Admin access
+
+- If enabled, provides administrative control over the dataset. That is, the ability to lock and unlock the dataset in addition to all of the above privileges. For more information on locking and unlocking, see here.
+
+# Applying Access Controls
+
+The aforementioned controls can be applied for individual datasets by those who have administration access to the dataset. By default, this is the owner of the dataset and Information System Admins.
+
+Each dataset can have a rule for its accessibility to (the group of all) general datastore users, and (optionally) extra rules for multiple custom groups of users. The following sections will discuss these.  
+
+## General user access to dataset
+
+### Who are general users?
+
+General Datastore users refers to the group of all users who can access the datastore. As such, general user access controls facilitate the general privacy or publicity (to all datastore users) of a dataset. You could limit access controls for general users if you wish to keep your dataset private to a group (perhaps during development processes and then later release it by updating general user access rights).
+
+### Where to manage general user access
+
+Upon registering a dataset, the general datastore user access rights can be managed by navigating to the 'settings' tab when viewing the dataset.
+
+In future versions of the IS, the ability to set access levels during the registration process will be implemented. For now, a workaround is to register a dataset with meaningless metadata, limit general access privileges, then meaningfully update the metadata as you wish.
+
+## Group access to dataset
+
+The group access feature enables custom permissions for groups (and thus members) to a dataset. This can achieve the privatisation of datasets to only certain users of the information system by *not* enabling general user read/write permissions and creating a group with enabled read/write permissions for its members.
+
+### Creating a group
+
+Group creation is currently performed by Information System Administrators. Please get in touch and we can help facilitate this for you.
+
+### Adding group access to datasets
+
+Groups access to datasets can be manually managed by navigating to the dataset's settings tab, then Access Control->Groups. Here, group permission can be added to the dataset. Group access levels for this dataset can also be edited here by selecting the group.
+
+Group access can be automatically applied with default values upon the creation of a new dataset. This occurs if 'Default data store access configuration' is enabled in the group creation process by the IS administrator. The result is that if a user belonging to that group creates a new dataset, then it will automatically be made available to the group according to the specified default access levels. Access levels can then be updated from default by navigating to the group in the dataset as mentioned in the above paragraph.
```

Comment:
Don't include this - I don't think we have the intention of configuring groups like this until the feature is more mature.

Reply:
removed line 97.

Reply:
Resolved